source/more-info/unsupported/privileged.markdown

```diff
@@ -15,3 +15,9 @@ If you are running an older version of our Home Assistant OS, update it in the
 
 If this is not our Home Assistant OS, you need to re-run our
 [convenience installation script](https://github.com/home-assistant/supervised-installer).
+
+You can also attempt to run the following command on the host:
+
+```bash
+sudo systemctl restart hassio-supervisor.service
+```
```

Comment:
How/Why would that solve an incorrectly configured permission?

Reply:
I'm not sure why it fixes it. I have the latest version of Home Assistant and rerunning the full installation script is not an option. I googled this error around and found this solution in a forum thread on the Home Assistant website. I tested it out and it fixed it.

I thought it would be useful for the next person who may run into this issue to try this step too.

I found the same command in multiple places:

- https://community.home-assistant.io/t/unable-to-update-supervisor-not-privileged/251511
- https://www.reddit.com/r/homeassistant/comments/ljtbdg/how_supervisor_becomes_not_privileged/

Reply:
Those refer all to Ubuntu, which is not a supported operating system for Home Assistant Supervised.

Reply:
Oh, sorry didn't know. I just clicked on the link from Home Assistant Supervisor page and it took me there. I decided to edit the file after finding a solution that worked for many others experiencing the same issue. I will update the text to indicate that it is for Ubuntu only.

Reply:
In that case, we won't be adding it, as Ubuntu is not supported.